### PR TITLE
Added hue scripts to the Cloudera/hue repo

### PR DESCRIPTION
**Pull request by pychheda**

### CODE REVIEW

File: tools/ops/script_runner/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Set of Hue scripts to help manage hue.  While some of these are self contained, the most important
```
Comment (romainr) [inline tools/ops/script_runner/README.md:1]: Maybe reformat a bit the README to be more readable?

e.g. 

# Ops Script
Set of ....

    git clone ...

**Review by romainr — APPROVED** [review]
Nice, just one main comment!

File: tools/ops/script_runner/README.md (hunk 1 above)
Comment (romainr) [reply tools/ops/script_runner/README.md:1]: And maybe renaming the commit `Add hue scripts from Chris Conner's personal repo` to something more generic
(I could squash all the commits when pushing too if sompler)

Comment (pychheda) [reply tools/ops/script_runner/README.md]: Done.

**Comment by romainr** [comment]
Nice!

Then we should skeep the CI on these, you can add in
https://github.com/cloudera/hue/blob/master/tools/ci/check_for_python_lint.sh#L27

    grep -v /tools/ops/ | \

And then should be all ready!

**Review by Akhilsnaik — APPROVED** [review]

**Comment by romainr** [comment]
Something like https://github.com/cloudera/hue/pull/1784#issuecomment-781449246 still needed for the CI

**Comment by romainr** [comment]
:+1: 
Should be good, but need to rebase the branch from master

**Comment by romainr** [comment]
Rebase on more time?

**Comment by pychheda** [comment]
Done!
Thanks, Romain.

**Pull request merged**